### PR TITLE
ci: fix dockerfile

### PR DESCRIPTION
**Pull request by cristianmtr**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,5 @@
-FROM jinaai/jina:2.0
+FROM jinaai/jina:2.0.8
+RUN apt-get update && apt-get install -y python3.7 python3.7-dev python3-pip git
```
Comment (tadejsv) [inline Dockerfile:2]: Why is python installed?

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,5 @@
-FROM jinaai/jina:2.0
+FROM jinaai/jina:2.0.8
```
Comment (bwanglzu) [inline Dockerfile:1]: let's use 2.0.10?

**Review by bwanglzu — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (cristianmtr) [reply Dockerfile]: this is not going to scale, to always update the version. Needs to be addressed with the versioning system currently in discussion

File: Dockerfile (hunk 1 above)
Comment (cristianmtr) [reply Dockerfile]: removed

**Review by tadejsv — APPROVED** [review]
LGTM

**Pull request merged**